### ISSUE TITLE
Visual indication of outcome

### ISSUE THREAD

**Issue by rogeralsing** [petabridge/NBench#65]
Instead of just reporting numbers or pass or fail.
It would be nice with some visual indication on the difference between the current and previous runs.
A way to see how much better or worse the the outcome was.
Different shades of gray/geen/red or arrows at different angles.

And possibly also some aggregated view/report where all outputs worth noticing is rendered.
Just show the significant changes there.

**Comment by pjquirk** [comment]
Yes!  Single-run visual summaries are helpful, but multi-run trends are even better.  I'd love it if this framework could be used to generate graphs for tracking performance changes, similar to Mozilla's [arewefastyet](https://arewefastyet.com/).
